packages/@storefront/core/test/unit/services/url.ts

```diff
@@ -347,6 +364,29 @@ suite('URL Service', ({ expect, spy, stub, itShouldBeCore, itShouldExtendBaseSer
       expect(handleUrl).to.be.calledOnce;
     });
 
+    it('should accept a url override', () => {
+      const currUrl  = '/foo';
+      const route = 'search';
+      const state = { a: 'b' };
+      const urlOverride = '/bar';
+      service['app'] = <any>{ flux: { store: { getState: () => state } } };
+      service['emitUrlUpdated'] = spy();
+      service['filterState'] = spy(() => state);
+      service['handleUrl'] = spy();
+      service['opts'] = { redirects: {} };
```

Comment:
Are these using `[]` because of TypeScript? If not, then just use `.`. Additionally, if you don't need to assert that these spies are called, then just replace them with `() => null`.

Reply:
Noted, will update.

Reply:
Majority of dynamic property accessors replaced with `.` notation. `['app']` and `['opts']` left for consistency with other tests.